src/test/java/Homework18.java

```diff
@@ -0,0 +1,21 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+    @Test
+    public void playSong() {
+        login("dmytro.kravchuk@testpro.io", "Fr440003");
+//        WebElement playSongButton = driver.findElement(By.cssSelector("[data-testid='play-btn']"));
+//     playSongButton.click();
+//WebElement nextSongButton = driver.findElement(By.cssSelector("[data-testid='play-next-btn']"));
+//nextSongButton.click();
+        WebElement allSongs = driver.findElement(By.cssSelector("[href='#!/songs']"));
+        allSongs.click();
+        WebElement  shuffleButton = driver.findElement(By.cssSelector("[data-test='btn-shuffle-all']"));
+        shuffleButton.click();
+WebElement soundBar = driver.findElement(By.cssSelector("[data-test='soundbars']"));
+        Assert.assertTrue(soundBar.isDisplayed());
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)